DFReader.py

```diff
@@ -59,7 +65,8 @@ def __init__(self, type, name, flen, format, columns):
                 msg_mults.append(mul)
                 msg_types.append(type)
             except KeyError as e:
-                raise Exception("Unsupported format char: '%s' in message %s" % (c, name))
```

Comment:
Stylistically I'm not in favour.  I'd rather have the longer line than create a variable, regardless of what pep says :-)

I'd also like to see the Exception narrowed to something else - ValueError if nothing else...